pkg/vulnsrc/debian/debian_test.go

```diff
@@ -104,6 +104,19 @@ func TestVulnSrc_Update(t *testing.T) {
 						FixedVersion: "19.4-2",
 					},
 				},
+				{
+					// Fix version not released yet
+					Key: []string{
```

Comment:
Can you also add `vulnerability-detail` and `vulnerability-id` buckets for CVE-2023-5981?

Reply:
@DmitriyLewen - done